hooks/990-ensure-uids-gids.chroot

```diff
@@ -24,13 +24,9 @@ echo "Ensure no docker"
 ! grep -q docker /etc/passwd /etc/shadow /etc/group /etc/gshadow
 rc=$?; [ "$rc" != "0" ] && MISMATCH=1
 
-echo "Ensure no user 107 (ex-docker)"
-cut -d: -f3 /etc/passwd | grep -q 107
-rc=$?; [ "$rc" = "0" ] && MISMATCH=1
+# uid 107 is used by rtkit
 
-echo "Ensure no group 113 (ex-docker)"
-cut -d: -f3 /etc/group | grep -q 113
-rc=$?; [ "$rc" = "0" ] && MISMATCH=1
+# gid 113 is used by pipewire
```

Comment:
We can leave these tests as is if we don't reuse these uids/gids.

Reply:
Done.

Reply:
Done.